GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -410,6 +410,12 @@ public void run() {
                     mGVRContext, "button/play-active.png"));
             mPlayPauseButton = new GVRSceneObject(gvrContext,
                     gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
+            mPlayPauseButton.getRenderData().getMaterial().setTexture("active_play", mActivePlay);
```

Comment:
Although I have been using this "pattern" to keep textures from recycling (most of the time until recently), isn't it more appropriate to use setKeepWrapper(true) on the texture instead of this roundabout way to add a reference?